docs/standard/microservices-architecture/net-core-single-containers-linux-windows-server-hosts/index.md

```diff
@@ -113,6 +113,8 @@ services:
   command: /bin/bash -c "dotnet restore ./eShopWeb.sln && dotnet publish  ./eShopWeb.sln -c Release -o ./obj/Docker/publish"
 ```
 
+[!INCLUDE[DotNet Restore Note](~/includes/dotnet-restore-note.md)]
```

Comment:
Here, rather than the include, insert the custom text, "Note:  Starting with **.NET Core 2.0**, the `dotnet restore` command executes automatically when `dotnet publish` is executed."